docs/your-first-script.md

```diff
@@ -2,115 +2,209 @@
 
 # Your first script
 
+Once you have Nextflow installed, you’re ready to run your first script. This guide details fundamental skills to run a basic Nextflow pipeline. It includes:
+
+- Running a pipeline
+- Modifying and resuming a pipeline
+- Configuring a pipeline parameter
+
+<h3>Prerequisites</h3>
+
+You will need the following to get started:
+
+- Nextflow 22.10.0 or later. See {ref}`install-page` for instructions to install or update your version of Nextflow.
```

Comment:
Any reason why you used 22.10 here? We typically don't provide ongoing support for versions older than a year.

Reply:
DSL1 > DSL2 and the need to enable DSL2 with `nextflow.enable.dsl=2`

I'm not committed to keeping it. I'll remove it.

Reply:
I wouldn't go any older than 23.10

Reply:
Updated 👍

Reply:
Sorry, I had some further comments I wanted to make but haven't had time. Will try to review today or tomorrow